src/main/java/de/learny/domain/Account.java

```diff
@@ -22,7 +22,7 @@
 	private String accountName;
 	private String password;
 	
-	@OneToMany
+	@OneToMany(mappedBy="account")
```

Comment:
mappedBy sorgt dafür, dass diese Verknüpfung von beiden Seiten aus erreichbar ist. Account weiß also welche testScores dazu gehören und testscore weiß zu welchem account sie gehören. In der Datenbank wird das aber nur in testScore.account eingetragen. JPA macht den Join dann automatisch. (Ohne mapped by entsteht noch eine tabelle die für jeden account die Tests beinhält... -> ist nicht nötig